docs/docs/optimizing-performance.md

```diff
@@ -160,17 +160,17 @@ You can learn more about this in [Webpack documentation](https://webpack.js.org/
 
 Remember that you only need to do this for production builds. You shouldn't apply `UglifyJsPlugin` or `DefinePlugin` with `'production'` value in development because they will hide useful React warnings, and make the builds much slower.
 
-## Profiling Components with Chrome Timeline
+## Profiling Components with Chrome Performance
```

Comment:
**Nitpick:**
"Profiling Components with Chrome Performance" reads funny to me. What about something like: "Profiling Components with the Chrome Performance Tab"